navigation/ca_swarm/config/global_costmap_params.yaml

```diff
@@ -0,0 +1,6 @@
+global_costmap:
+  global_frame: /map
```

Comment:
@dmnunez1993 switch to `map`, see https://github.com/RoboticaUtnFrba/create_autonomy/pull/115#discussion_r343733343.